pkg/converter/convert.go

```diff
@@ -27,6 +27,8 @@ const (
 	argUsername     = "--username"
 	argPassword     = "--password"
 	argLoginMethod  = "--login"
+	argConfigMode   = "--" + cfgConfigMode
+	argApiserverID  = "--" + cfgApiserverID
```

Comment:
do you still need these?

Reply:
yes, for these: 
`argApiserverIDVal := getExecArg(authInfo, argApiserverID)`
`argConfigModeVal := getExecArg(authInfo, argConfigMode)`

Reply:
but exec formatted kubeconfig doesn't use `--config-mode` nor `apiserver-id`